src/Symfony/Component/Messenger/Tests/Command/FailedMessagesRetryCommandTest.php

```diff
@@ -11,14 +11,18 @@
 
 namespace Symfony\Component\Messenger\Tests\Command;
 
+use Exception;
```

Comment:
We don't use `use` for PHP built-on classes. Can you use `\Exception` in the code instead?

Reply:
Thanks for reviewing!